packages/file_selector/file_selector_platform_interface/lib/src/method_channel/method_channel_file_selector.dart

```diff
@@ -93,4 +93,18 @@ class MethodChannelFileSelector extends FileSelectorPlatform {
       },
     );
   }
+
+  /// Gets a list of directories paths from a dialog
```

Comment:
Overrides don't need to repeat the comment (feel free to remove the others from this file for consistency.